tools/fpgabist/bist_common.py

```diff
@@ -78,6 +80,13 @@ def get_mode_from_path(gbs_path):
     return None
 
 
+def is_command_in_path(cmd):
+    for dir in os.getenv("PATH").split(':'):
+        if (os.path.exists(os.path.join(dir, cmd))):
+            return True
+    return False
```

Comment:
If you'll change your function to `find_executable` this won't be needed, although instead of having two `return`s you could do:
```
checked_paths = [os.path.exists(os.path.join(directory, cmd)) for directory in os.getenv("PATH").split(":")]
return any(checked_paths)
```
```
Help on built-in function any in module __builtin__:

any(...)
    any(iterable) -> bool

    Return True if bool(x) is True for any x in the iterable.
    If the iterable is empty, return False.
```